pkgs/by-name/mu/muvm/package.nix

```diff
@@ -55,13 +55,13 @@ rustPlatform.buildRustPackage rec {
   ];
 
   postFixup = ''
-    wrapProgram $out/bin/krun $wrapArgs \
+    wrapProgram $out/bin/muvm $wrapArgs \
       --set-default OPENGL_DRIVER ${opengl-driver}
```

Comment:
I don't think this does anything without https://github.com/AsahiLinux/muvm/pull/38.

Reply:
Done